enterprise/frontend/src/embedding-sdk/components/private/PublicComponentWrapper/SdkError.tsx

```diff
@@ -2,20 +2,25 @@ import { t } from "ttag";
 
 import { useSdkSelector } from "embedding-sdk/store";
 import { getErrorComponent } from "embedding-sdk/store/selectors";
+import { Center } from "metabase/ui";
 
-export type SdkErrorProps = { message: string };
+export type SdkErrorProps = { message: string; className?: string };
 
-export const SdkError = ({ message }: SdkErrorProps) => {
+export const SdkError = ({ message, className }: SdkErrorProps) => {
   const CustomError = useSdkSelector(getErrorComponent);
 
   if (CustomError) {
     return <CustomError message={message} />;
   }
 
   return (
-    <div>
-      <div>{t`Error`}</div>
-      <div>{message}</div>
-    </div>
+    <Center className={className} h="100%" w="100%" mx="auto">
+      {CustomError || (
+        <div>
+          <div>{t`Error`}</div>
+          <div>{message}</div>
+        </div>
+      )}
+    </Center>
```

Comment:
Does the "Center" component there also enforces other CSS properties, such as text alignment and flex alignment? I worry that it would interfere with the SDK user's error style, e.g. if they want a different alignment. If not, then this is fine.

(this also applies to SdkLoader below)

Reply:
nice!